components/calendar/calendar.ts

```diff
@@ -221,13 +223,20 @@ export class Calendar implements AfterViewInit,OnInit,OnDestroy,ControlValueAcce
     
     @Output() onSelect: EventEmitter<any> = new EventEmitter();
     
-    @Input() locale: LocaleSettings = {
+    @Input() locale: LocaleSettings = !moment ? {
```

Comment:
This will cause a `ReferenceError`. Do something like `typeof moment === 'undefined'` instead.